shell/platform/linux/fl_gesture_helper.h

```diff
@@ -0,0 +1,111 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_LINUX_FL_GESTURE_HELPER_H_
+#define FLUTTER_SHELL_PLATFORM_LINUX_FL_GESTURE_HELPER_H_
+
+#include <gtk/gtk.h>
+
+#include <glib-object.h>
+
+G_BEGIN_DECLS
+
+G_DECLARE_FINAL_TYPE(FlGestureHelper,
+                     fl_gesture_helper,
+                     FL,
+                     GESTURE_HELPER,
+                     GObject)
+
+/**
+ * FlGestureHelper:
+ *
+ * #FlGestureHelper captures and redistributes mouse events to embedded platform
+ * views.
+ */
+
+/**
+ * fl_gesture_helper_new:
+ *
+ * Creates a new #FlGestureHelper object.
+ *
+ * Returns: a new #FlGestureHelper.
+ */
+FlGestureHelper* fl_gesture_helper_new();
+
+/**
+ * fl_gesture_helper_button_press:
+ * @gesture_helper: an #FlGestureHelper.
+ * @event: a #GdkEventButton event.
+ *
+ * Redistrubutes @event to a embedded platform view.
+ */
+void fl_gesture_helper_button_press(FlGestureHelper* gesture_helper,
```

Comment:
I don't see usage for a few methods in the gesture handler, are they currently unused?

Reply:
used in fl_view.cc